LFS171x/composer-material/tuna-network/queries.qry

```diff
@@ -0,0 +1,7 @@
+query getTunaByParticipant {
```

Comment:
Does .qry file allow comments? If so, can we add an SPDX license line to reflect Apache 2.0 license? Similar to:
`// SPDX-License-Identifier: Apache-2.0`